_stbt/control_gpl.py

```diff
@@ -120,7 +121,14 @@ class HdmiCecControl(object):
     }
 
     def __init__(self, device, source, destination):
-        import cec
+        # On Ubuntu 18.04 `cec/__init__.py` tries to import `_cec` but can't
+        # find it.
+        sys.path.insert(0, "/usr/lib/python2.7/dist-packages/cec")
+        try:
+            import cec
+        finally:
+            sys.path.pop(0)
```

Comment:
I've submitted this bug upstream: https://bugs.launchpad.net/ubuntu/+source/libcec/+bug/1805620